client/routes/execution/event-details.vue

```diff
@@ -0,0 +1,139 @@
+<script>
+import moment from 'moment'
+import shortName from '../../short-name'
+import parentWorkflowLink from './parent-workflow-link'
+
+const eventOneLiners = {
+  WorkflowExecutionStarted: d => {
+    var summary = {
+      Parent: undefined,
+      input: d.input,
+      identity: d.identity,
+      Workflow: shortName(d.workflowType.name),
+      'Close Timeout': moment.duration(d.executionStartToCloseTimeout, 'seconds').format()
+    },
+    wfLink = parentWorkflowLink(d)
+
+    if (wfLink) {
+      summary.Parent = { routeLink: wfLink.to, text: wfLink.text }
+    }
+    return summary
+  },
+  StartChildWorkflowExecutionInitiated: d => ({
+    Workflow: shortName(d.workflowType.name),
+    Tasklist: d.taskList.name,
+    input: d.input
+  }),
+  ChildWorkflowExecutionStarted: d => ({
+    Workflow: {
+      routeLink: {
+        name: 'execution/summary',
+        params: {
+          domain: d.domain,
+          workflowId: d.workflowExecution.workflowId,
+          runId: d.workflowExecution.runId
+        }
+      },
+      text: `${shortName(d.workflowType.name)} - ${d.workflowExecution.workflowId}`
+    }
+  }),
+  ChildWorkflowExecutionCompleted: d => ({
+    Workflow: {
+      routeLink: {
+        name: 'execution/summary',
+        params: {
+          domain: d.domain,
+          workflowId: d.workflowExecution.workflowId,
+          runId: d.workflowExecution.runId
+        }
+      },
+      text: shortName(d.workflowType.name)
+    },
+    result: d.result
+  }),
+  DecisionTaskScheduled: d => ({
+    Tasklist: d.taskList.name,
+    Timeout: moment.duration(d.startToCloseTimeoutSeconds, 'seconds').format()
+  }),
+  DecisionTaskStarted: d => ({ requestId: d.requestId }),
+  DecisionTaskCompleted: d => ({ identity: d.identity }),
+  ActivityTaskScheduled: d => ({
+    ID: d.activityId,
+    Name: shortName(d.activityType.name),
+    input: d.input,
+    'Close Timeout': moment.duration(d.scheduleToCloseTimeoutSeconds, 'seconds').format()
+  }),
+  ActivityTaskStarted: d => ({ identity: d.identity, requestId: d.requestId }),
+  ActivityTaskCompleted: d => ({ result: d.result })
```

Comment:
do we plan to add other events like Signal, Markers (for version, side effect, local activity), etc.